<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriversWidgetClass</class>
 <widget class="QWidget" name="DriversWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DriversWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="reportsListWidget"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="descriptionLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Coordinates</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="coordsLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="chatRoom"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="messageLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="locationLabel">
         <property name="text">
          <string>//for locaion</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Score:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="scoreLabel">
         <property name="text">
          <string>//for score</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
